onchain/rollups/test/foundry/util/LibServerManager.sol

```diff
@@ -0,0 +1,184 @@
+// (c) Cartesi and individual authors (see AUTHORS)
```

Comment:
My intention was to empty the `dapp/helper` directory a little bit.
Also, because this library is not generated by some script, we can place it anywhere we'd like.